src/Benchmarks/Benchmarks.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <!-- For local dev specify all supported TFMs they can be easily tested -->
   <PropertyGroup Condition="'$(BenchmarksTargetFramework)' == ''">
-    <TargetFrameworks>net7.0</TargetFrameworks>
```

Comment:
I'd rather not change that because when users try to build it locally it's easier to just require a tfm that is probably installed locally.